continuous-pipe.yml.erb

```diff
@@ -5,7 +5,9 @@ variables:
   CP_ENVIRONMENT:
     name: CP_ENVIRONMENT
     expression: '"<%= config.name %>-" ~ code_reference.branch'
-  CP_ENVIRONMENT_DOMAIN_SUFFIX: -<%= config.name %>.webpipeline.net
+  CP_ENVIRONMENT_DOMAIN:
+    name: CP_ENVIRONMENT_DOMAIN
+    expression: 'slugify(hash_long_domain_prefix(code_reference.branch, 39)) ~ "-<%= config.name %>.webpipeline.net"'
```

Comment:
Not sure if we can leave this as 39 as it depends on the length of the config.name value. I think the limit is 64 characters?

Reply:
sorry I just blindly copied it from where you wrote it for a project. I suppose we could move the config.name to another var and pull that in to find length without duplicating the value (if CP correctly does that)

Reply:
Fixed the only way I saw possible without updating CP itself, ruby finding the string length in the seed templating process.